Sources_v3/StreamChat/ChatClient.swift

```diff
@@ -250,7 +241,7 @@ public class _ChatClient<ExtraData: ExtraDataTypes> {
         } else {
             workerBuilders = []
             eventWorkerBuilders = []
-            environment.webSocketClientBuilder = { _, _, _, _, _, _ in nil }
+            environment.webSocketClientBuilder = { _, _, _, _, _ in nil }
```

Comment:
I just realized, imagine `webSocketClientBuilder` being optional instead of the returning value being optional 🤯

Reply:
fixed in c732d658ca9625bd74d35c32c4d80b3a70871a0e